src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/AssertExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using Xunit;
+using Xunit.Sdk;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions
+{
+    public static class AssertExtensions
+    {
+        public static T AssertFail<T>(string message) 
```

Comment:
I think it would be good to add a comment to say how this is intended to be used.

Reply:
Added a comment